forte/data/data_pack.py

```diff
@@ -244,7 +243,7 @@ def text(self) -> str:
     @property
     def audio(self) -> Optional[np.ndarray]:
         r"""Return the audio of the data pack"""
-        return self.get_payload_data_at(Modality.Audio, 0)
+        return cast(np.ndarray, self.get_payload_data_at(Modality.Audio, 0))
```

Comment:
Should write down an issue saying this one is a temporary solution to return the payload data (combining with the lazy loading and payload we can make this more reliable)